subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -7974,7 +7974,7 @@ u32_t radio_adv_enable(u16_t interval, u8_t chl_map, u8_t filter_policy)
 #if defined(CONFIG_BLUETOOTH_CONTROLLER_DATA_LENGTH)
 		conn->default_tx_octets = _radio.default_tx_octets;
 		conn->max_tx_octets = RADIO_LL_LENGTH_OCTETS_RX_MIN;
-		conn->max_rx_octets = RADIO_LL_LENGTH_OCTETS_RX_MIN;
+		conn->max_rx_octets = CONFIG_BLUETOOTH_CONTROLLER_DATA_LENGTH_MAX;
```

Comment:
Revert this, the default connection Rx length is 27 and in line 1625, after DLE procedure the max_rx_octets will be increased. Why? Don't want the Radio listening longer and wasting current if packets have CRC error and length used by Radio is incorrectly long (compared to default 27 bytes).